docs/migration.md

```diff
@@ -879,13 +882,7 @@ receivers:
     add_attributes: true
 
 processors:
-  ## There is no substitute for `Description` in current project phase.
-  ## It is recommended to use comments for that purpose, like this one.
-  ## sumologic_syslog/<source group name>:
-  ## <source group name> can be substitute of Installed Collector `Name`.
-  sumologic_syslog/syslog source:
-
-  ## The following configuration will add two fields to every record
+  ## The following configuration will add two fields to every resource
```

Comment:
I would leave this processor, as it ensures backward compatibility with syslog source
https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/sumologicsyslogprocessor

Please ensure that final configuration of OTC produces the same result as Installed Collector

Reply:
I tested this with the `syslog exporter`, I can double check it with the `sumologic exporter` again

Reply:
The OT collector registered successfully using this endpoint on long: https://long-open-events.sumologic.net

I saw the log message here:
https://stagdata.long.sumologic.net/ui/#/search/create?id=1ML0NOm6P7QbVhNPtWxBeunSJUhINTnfEapIjTWi

The only abnormality I noticed was that the `source name` and `source host` were `undefined`

Reply:
We need to fix it. `facility` is probably due to missing `sumologicsyslogprocessor`

Reply:
`net.peer.name` is probably due to `tcp`/`udp` configuration

Reply:
used `'groupbyattr` to fix this.